src/sundials/shared/sundialsmath.c

```diff
@@ -79,7 +85,8 @@ realtype RAbs(realtype x)
 realtype RPower2(realtype x)
 {
 #if defined(SUNDIALS_USE_GENERIC_MATH)
-  return((realtype) pow((double) x, 2.0));
+printf("RPower2\n");
```

Comment:
Looks like it could be a leftover from debugging.

Reply:
removed the printf.